### PR TITLE
routes resources and book , sponsor and attend paths

### PR DESCRIPTION
**Pull request by olijvecaroline**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,3 +1,19 @@
...
+
+
+    resources :events do  sponsors
+                          companies
+                          workshops do users
+                          end
+                          end
+    "event/book", to: "event#book"
+    "event/sponsor" to: "event#sponsor"
+    "event/workshop/attend" to: "workshop/attend"
```
Comment (Woulei) [inline config/routes.rb]: Maybe change to something like this:

"event/:id/attend" to: "event#attend" as "attend_path", method: :get <-- to render attend form
"event/:id/attend"  to: "event#attend" as "attend_path, method: :post <-- to save the attend form

File: config/routes.rb [hunk 2]
```diff
@@ -1,3 +1,19 @@
...
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  root to: 'pages#home'
+
+
+
+    resources :events do  sponsors
+                          companies
+                          workshops do users
+                          end
```
Comment (Woulei) [inline config/routes.rb]: don't think these should be nested.
Also let devise handle the user routes.

Comment (arnoFleming) [inline config/routes.rb]: this is the same content as in PR #31 ... Why this duplication?

Comment (arnoFleming) [inline config/routes.rb]: ping

Comment (V-Ho) [inline config/routes.rb]: I'm not sure. It doesn't make sense for the changes to the routes to be in the same pull request as the upcoming events view changes. I propose that I remove the routes changes from that pull request, and this one can focus on just the routes. Does that sound okay?

**Comment by anikaSchwing** [comment]
@arnoFleming Please review this?

**Pull request closed without merging**